contrib/devtools/update-translations.py

```diff
@@ -122,6 +122,12 @@ def escape_cdata(text):
     text = text.replace('"', '&quot;')
     return text
 
+def check_for_address(s):
+    #checks each message for bitcoin addresses
+    address_rgx='(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,34}'
+    return bool(re.search(address_rgx,s))
```

Comment:
Probably fine to just inline (using the compiled re constant), since it is only used once